app/reducers/index.js

```diff
@@ -0,0 +1,15 @@
+// @flow
```

Comment:
This file too is sus. I think there were changes in this file to add the pyodide reducer, but we've since update how we use Redux. Will probably have to follow the new, existing logic.